src/components/ThankYouPage.tsx

```diff
@@ -0,0 +1,36 @@
+import { useTranslation } from "react-i18next";
+
+const ThankYouPage = () => {
+  const { t } = useTranslation();
+  const linkClass = "font-bold text-[#4E4E4E] text-base sm:text-lg md:text-xl lg:text-2xl mb-8 px-6 py-3 underline hover:opacity-80 transition";
+
+  return (
+    <div className="mt-10 flex flex-col items-center justify-center w-full gap-y-6 px-4">
+      <h1 className="font-bold text-[#4E4E4E] text-center text-[clamp(2rem,5vw,3.125rem)] max-w-[90%] sm:max-w-[75%] md:max-w-[50%] mx-auto">
+        {t("repeat-instruction")}
+      </h1>      
+      <div className="flex flex-col sm:flex-row flex-wrap justify-center gap-2 sm:gap-4">
+        <a href="https://www.thetrevorproject.org/breathing-exercise/"
```

Comment:
I think this button should redirect to the "/" of the app, no to the trevor website